tests/tests/uia_tests.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015 Cryptonomex, Inc., and contributors.
+ * Copyright (c) 2018 Cryptonomex, Inc., and contributors.
```

Comment:
2015, or 2015-2018.

Reply:
changed that, sorry https://github.com/bitshares/bitshares-core/pull/705/commits/fb1d0be0f4d823a9430a59a5177bcbb860e310ea